### DISCUSSION TITLE
Support for prophet

### DISCUSSION THREAD

**Discussion by fsty** [autogluon/autogluon#2975] [Q&A] [answered]
Hi there

There is any plan to add support for prophet as local model in time series forecasting ?

**Comment by shchur** [comment, marked as answer]
Hi! Prophet was included in an earlier release of AutoGluon but we removed it because (1) its dependencies introduced problems on some operating systems, (2) the runtime was quite slow and (3) it didn't result in a noticeable accuracy improvement on the benchmark datasets that we considered
